### PR TITLE
testing

### PR DESCRIPTION
**Pull request by robinagandhi**
this is for test

### CODE REVIEW

File: module-1/github-primer.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ After finishing the hello-world tutorial, you will have created a **Remote** rep
 ### Step 3
 To create a **Local** repository there are two basic options.
 1. Clone a remote repository on your computer, or
-2. Initialize a new git repository from scratch on your computer.
+2. Initialize a new git repository from scratch on you
```
Comment (robinagandhi) [inline module-1/github-primer.md:42]: seems like you missed something

**Comment by robinagandhi** [comment]
i dont like it.

**Comment by MLHale** [comment]
I don't like it either!

**Comment by robinagandhi** [comment]
@MLHale: This was part of the github demonstration at the midwestbigdata hub all-hands meeting. I used the github primer for it. It was a small audience but the content was well received. :-)

**Comment by MLHale** [comment]
@robinagandhi I figured it was, I got an email from github and figured I would say something to give you another talking point about the collaborative nature of pull requests.

Good that your discussion was well received!

**Pull request closed without merging**